doc/main.md

```diff
@@ -13,6 +13,10 @@ The extension can function in one of two modes:
 
 Each mode is described in detail on its own page.
 
+Furthermore, the extension provides:

```

Comment:
On line 20, you repeat yourself with another sentence beginning with "Furthermore`. Try to avoid repetition. The use of "Furthermore" tends to only be used once to add additional reinforcement to something, in this case, the features of the extension.